sources/assets/shells/aliases.d/uploader

```diff
@@ -0,0 +1 @@
+alias uploader='python3 /opt/tools/uploader/uploader.py'
```

Comment:
You must use the python binary venv : `alias uploader='/opt/tools/uploader/venv/bin/python3 /opt/tools/uploader/uploader.py'`

Reply:
done